integration_tests/pages/overview/EducationAndTrainingPage.ts

```diff
@@ -111,9 +110,9 @@ export default class EducationAndTrainingPage extends Page {
     return Page.verifyOnPage(AdditionalTrainingPage)
   }
 
-  clickToChangeEducationalQualifications(): QualificationsListPage {
+  clickToChangeEducationalQualifications<T extends Page>(constructor: new () => T): T {
```

Comment:
I had to make this change because clicking the Change link now wont always return the Qualifications List page as it used to.